blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sourcegraph 3.31 release"
+publishDate: 2021-08-20T10:00-07:00
+description: "Sourcegraph 3.31 introduces UPDATE"
```

Comment:
@iskyOS - can you update this description? I usually mention whatever is included in the highlights.